app/mailers/user_mailer.rb

```diff
@@ -13,6 +13,7 @@ def sign_slide_approved(args)
     @approver     = args[:approver]
     @sign         = args[:sign_slide].sign
     @slide        = args[:sign_slide].slide
+    @message      = args[:message] unless args[:message].blank?
```

Comment:
I don't think you need this `unless`.  Just check for `@message.blank?` in the view.  (since you are already checking for falsiness there anyway)